using-kubecost/navigating-the-kubecost-ui/budgets.md

```diff
@@ -60,3 +60,7 @@ Selecting _Edit_ in the row of a specific budget will open a window allowing you
 ### Deleting a budget
 
 Selecting _Delete_ will open the Delete Budget window. Confirm by selecting _Delete_.
+
+## Use cases
+
+For how to use budgets in a use case preventing cost overruns before they occur, see the guide [here](/using-kubecost/proactive-cost-controls.md).
```

Comment:
wording a tad awkward. Since this is under 'use cases' i'd suggest simply saying: "Enforce Kubecost Budgets to prevent overruns before they occur" -- you could maybe make the whole use case link to the guide

Reply:
Like this?